api/src/Entity/MaterialList.php

```diff
@@ -147,6 +147,13 @@ public function getName(): ?string {
             ?? 'NoName';
     }
 
+    #[ApiProperty(example: 3)]
+    #[SerializedName('itemCount')]
+    #[Groups(['read'])]
+    public function getItemCount(): ?int {
```

Comment:
```suggestion
    public function getItemCount(): int {
```

When should the number of items be null?